network/peers/connections/filters_test.go

```diff
@@ -1,24 +1,167 @@
 package connections
 
 import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
 	"testing"
+	"time"
 
+	"github.com/bloxapp/ssv/logging"
 	"github.com/bloxapp/ssv/network/records"
+	"github.com/bloxapp/ssv/utils/rsaencryption"
+	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	SenderPrivateKey *rsa.PrivateKey
+
+	HandshakeData       records.HandshakeData
+	HashedHandshakeData []byte
+	Signature           []byte
+
+	SenderPeerID    = peer.ID("1.1.1.1")
+	RecipientPeerID = peer.ID("2.2.2.2")
+
+	PrivateKeyPEM      []byte
+	SenderPublicKeyPEM []byte
+)
+
+func init() {
+	var err error
```

Comment:
I rather you take this out to a function and call it at the beginning of every test.

Reply:
done